fritz

```diff
@@ -21,24 +23,62 @@ def status(message):
         print(f'\r[✓] {message}')
 
 
-def check_configs(cfgs):
-    # use config defaults if configs do not exist
-    for cfg in cfgs:
-        # cwd = str(Path(cfg).parent)
-        c = cfg.replace('.defaults', '')
-        if not Path(c).exists():
-            cd = input(f'{c} does not exist, do you want to use {cfg} (not recommended)? [y/N] ')
-            if cd.lower() == 'y':
-                s = subprocess.run(["cp", f"{cfg}", f"{c}"], check=True)
-                if s.returncode != 0:
-                    raise IOError(f"Failed to copy {cfg} to {c}")
-            else:
-                raise IOError(f'{c} does not exist, aborting')
+def check_config(cfg='fritz.defaults.yaml'):
+    c = cfg.replace('.defaults', '')
+    if not Path(c).exists():
+        cd = input(f'{c} does not exist, do you want to use {cfg} (not recommended)? [y/N] ')
+        if cd.lower() == 'y':
+            s = subprocess.run(["cp", f"{cfg}", f"{c}"], check=True)
+            if s.returncode != 0:
+                raise IOError(f"Failed to copy {cfg} to {c}")
+        else:
+            raise IOError(f'{c} does not exist, aborting')
+
+    # generate a token for SkyPortal to talk to Kowalski:
+    with open(c) as cyaml:
+        config = yaml.load(cyaml, Loader=yaml.FullLoader)
+
+    kowalski_token = generate_kowalski_token(
```

Comment:
Ideally move the function definition earlier on (forward references can be quite confusing while reading the code).

Reply:
Done.